### DISCUSSION TITLE
fixed the buypass

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#873] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://playful-meringue-fb58bc.netlify.app/
simple fix in code and host on netlify instead of vercel

**Comment by AshtonDavies** [comment]
Is this independent from Bypassi's page, or does it connect to and use his json files?

**Reply by S-PScripts** [reply]
i cloned in netlify then edited the files on github. then i deleted the github (but the netlify link still works)

**Reply by AshtonDavies** [reply]
Okay, then it does not work for people with the original one blocked. I was hoping it would be, so I could figure out how to make it run independently on other sites.

**Reply by tuta-amb** [reply]
@AshtonDavies What? Why would it not work? It is not connected to the original Buypass. Anyhow, hosting your own version is as simple as forking and publishing.

**Reply by S-PScripts** [reply]
I have realised that my filter can still block the same stuff that's blocked normally. For example, trying to go to discord makes the window error up saying that there was an error processing the order. 
But that probably doesn't happen for most filters i hope

**Reply by tuta-amb** [reply]
@S-PScripts Discord works great for me, using Lightspeed.

**Reply by AshtonDavies** [reply]
@tuta-amb If you just copy the HTML over, it will use the json files from bypassi.com, which is blocked for some users.

**Reply by tuta-amb** [reply]
@AshtonDavies Ofc it will, you have to copy everything over and change some paths too.

**Reply by S-PScripts** [reply]
no it doesn't, nothing from bypassi.com was in the repo

**Reply by tuta-amb** [reply]
@S-PScripts bro you absolute skid what are you yapping about

the repo was **DELETED**

**Reply by S-PScripts** [reply]
i deleted it cuz i didn't want it on my profile.
i meant that files from the bypassi website dont directly go to the clone i made, its's just from the repo

**Reply by tuta-amb** [reply]
@S-PScripts private repos are a thing? also your second sentence makes no sense

**Reply by AshtonDavies** [reply]
@tuta-amb Yes they are. You can choose the private option when creating a repo.

**Reply by S-PScripts** [reply]
@tuta-amb yeah i know that they exist but i deleted it anyway because it was fixed and it was mainly just an english version of https://sort.darkshade.design/ tbh. the files from bypassi don't directly get hosted as in my website is not just an embed of his
